### DISCUSSION TITLE
More languages out of the box?

### DISCUSSION THREAD

**Discussion by mauricioszabo** [pulsar-edit/.github#153] [Core Application & Packages]
Now that we're almost done with some tree-sitter modernization, I feel it's becoming easier to add more grammars, so maybe we can discuss bundling more languages with Pulsar?

I think some can be really useful for day to day work:

1. Dockerfile
2. Cleanup Javascript into only "Javascript", not "Babel" and "Javascript + React"
3. JSON 5 (JSON with comments)
4. Scheme (because tree-sitter queries are written on a dialect of scheme, and seems weird for they to not be highlighted)
5. Kotlin
6. Swift and ObjectiveC (maybe, for iOS stuff)

WDYT?

**Comment by Daeraxa** [comment]
Just gonna drop this here, don't mind me... :)

https://github.com/orgs/pulsar-edit/discussions/32

**Comment by confused-Techie** [comment]
Just [posted](https://github.com/orgs/pulsar-edit/discussions/32#discussioncomment-5388116) in continuing this topic, because @mauricioszabo total agree we should include more languages by default, given the specifics I mention there
